extensions/ql-vscode/src/vscode-tests/no-workspace/data/remote-queries/queries/MRVA Integration test 2-UL-vbKAjP8ffObxjsp7hN/query-result.json

```diff
@@ -0,0 +1,18 @@
+{
+  "executionEndTime": 1645645150738,
+  "analysisSummaries": [
+    {
+      "nwo": "github/vscode-codeql",
+      "resultCount": 5,
+      "fileSizeInBytes": 81237,
+      "downloadLink": {
+        "id": "171544171",
+        "urlPath": "/repos/dsp-testing/qc-run2/actions/artifacts/171544171",
```

Comment:
Perhaps shouldn't use `dsp-testing` testing?

Reply:
I can change the repo.